src/utils/storage-util.ts

```diff
@@ -0,0 +1,19 @@
+import { MMKV } from 'react-native-mmkv';
```

Comment:
Instead of `storage-utils`, create a custom hook `useLocalStorage` and have it expose these methods be more `reacty`

Also refine return type of the methods

Reply:
created a custom hook of local storage and used it, also refined the return type